ssl/build.info

```diff
@@ -1,11 +1,7 @@
-SUBDIRS=record rio
+SUBDIRS=record rio quic
 
 LIBS=../libssl
 
-IF[{- !$disabled{quic} -}]
-  SUBDIRS=quic
-ENDIF
-
```

Comment:
Does it mean that QUIC is no longer disablable?

Reply:
No. It just means that we've changed how that disabling is done (see ssl/quic/build.info). Some files are not disabled in a no-quic build. Specifically you can still use the 3rd party QUIC stack API even in the case where OpenSSL's own QUIC stack has been disabled.